### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by smirnovaka**

### CODE REVIEW

File: src/foam/dashboard/view/DashboardCitationView.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      class: 'Class',
+      name: 'of'
+    }
+  ],
+
+  methods: [
+    function initE() {
+      var self = this;
+      this
+      .on('click', function() {
```
Comment (kgrgreer) [inline src/foam/dashboard/view/DashboardCitationView.js]: indent this .on() block.

File: src/foam/dashboard/view/Table.js [hunk 2]
```diff
@@ -12,33 +12,46 @@ foam.CLASS({
...
-
   properties: [
+    {
+      name: 'citationView',
+      factory: function() {
+        return this.DashboardCitationView;
+      }
+    },
+    {
+      name: 'listDaoName'
```
Comment (kgrgreer) [inline src/foam/dashboard/view/Table.js]: listDAOName or just daoName or daoKey

File: src/foam/dashboard/view/Table.js [hunk 3]
```diff
@@ -57,10 +71,34 @@ foam.CLASS({
...
+              listDaoName: self.listDaoName
             }));
           })
-          return this.TableView.create({ data: dao });
-        }));
+          return self.E()
+            .addClass(this.myClass())
+            .select(dao, function(obj) {
+              return self.E()
+                .start().addClass('table-row')
```
Comment (kgrgreer) [inline src/foam/dashboard/view/Table.js:85]: You have two start()s but only one end().

File: src/foam/counter/Counter.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.counter',
+  name: 'Counter',
+
+  documentation: `Generic model that can be used by multiples daos for reporting purposes(e.g. groupby).`,
```
Comment (kgrgreer) [inline src/foam/counter/Counter.js:11]: I don't understand, but you can show/explain this to me.

File: src/foam/dashboard/view/DashboardCitationView.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+  methods: [
+    function initE() {
+      var self = this;
+      this
+        .on('click', function() {
+          self.openFilteredListView(self.data);
+        })
+        .addClass(this.myClass())
+        .start()
+          .addClass('id')
```
Comment (kgrgreer) [inline src/foam/dashboard/view/DashboardCitationView.js]: Does this work? Does the CSS get applied? Do you need to do this.myClass('id')? Same below.

File: src/foam/dashboard/view/Table.js (hunk 3 above)
Comment (kgrgreer) [reply src/foam/dashboard/view/Table.js:85]: indentation is off

File: src/foam/dashboard/view/DashboardCitationView.js (hunk 5 above)
Comment (smirnovaka) [reply src/foam/dashboard/view/DashboardCitationView.js]: not applied either way..

File: src/foam/dashboard/view/DashboardCitationView.js [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+  css: `
+    ^ {
+      display: flex;
+      justify-content: space-between;
+      padding-top: 15px;
+      padding-bottom: 15px;
+      border-bottom: 1px solid #e4e3e3;
+    }
+
+    ^.foam-dashboard-view-DashboardCitationView-id {
```
Comment (kgrgreer) [inline src/foam/dashboard/view/DashboardCitationView.js]: I think just ^id

File: src/foam/dashboard/view/DashboardCitationView.js [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+  package: 'foam.dashboard.view',
+  name: 'DashboardCitationView',
+  extends: 'foam.u2.View',
+
+  axioms: [
+    foam.pattern.Faceted.create()
+  ],
+
+  imports: [
+  'stack'
```
Comment (kgrgreer) [inline src/foam/dashboard/view/DashboardCitationView.js]: indent

File: src/foam/dashboard/view/DashboardCitationView.js [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+        .end()
+        .start()
+          .addClass(this.myClass('value'))
+          .add(this.data['value'])
+        .end()
+    },
+
+    function openFilteredListView(obj) {
+      var dao = this.__subContext__[obj.listDAOName].where(this.EQ(obj.searchKey, obj.id));
+      var config = foam.comics.v2.DAOControllerConfig.create({ dao: dao, hideQueryBar: false });
```
Comment (kgrgreer) [inline src/foam/dashboard/view/DashboardCitationView.js]: Better to require and then create as this.DAOControllerConfig.create()

File: src/foam/dashboard/view/DashboardCitationView.js (hunk 6 above)
Comment (kgrgreer) [reply src/foam/dashboard/view/DashboardCitationView.js]: This didn't work?

**Pull request merged**